packages/playground/src/portal/transfer_view.vue

```diff
@@ -19,6 +19,7 @@
                 :rules="[
                   validators.required('Recepient Twin ID is required'),
                   validators.isNotEmpty('Invalid Twin ID'),
+                  validators.isNumeric('Invalid Twin ID'),
```

Comment:
can we check also it's larger than 0

Reply:
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/3351e0cf-1c6d-4be7-93f3-eb28bdb37861)